docs/how-to-guides/block-tutorial/extending-the-query-loop-block.md

```diff
@@ -154,16 +160,16 @@ Even with all of this, your custom post type might have unique requirements: it
 
 ### Disabling irrelevant or unsupported query controls
 
-Let's say you don't use at all the `sticky` attribute in your books, so that would be totally irrelevant to the customization of your block. In order to not confuse the users as to what a setting might do, and only exposing a clear UX to them, we want this control to be unavailable. Furthermore, let's say that you don't use the `author` field at all, which generally indicates the person who has added that post to the database, instead you use a custom `bookAuthor` field. As such, not only keeping the `author` filter would be confusing, it would outright break your query.
+Let's say you don't use at all the `sticky` attribute in your books, so that would be totally irrelevant to the customization of your block. In order to not confuse the users as to what a setting might do, and only exposing a clear UX to them, we want this control to be unavailable. Furthermore, let's say that you don't use the `author` field at all, which generally indicates the person who has added that post to the database, instead you use a custom `bookAuthor` field. As such, not only keeping the `author` filter would be confusing, it would outright 'break' your query.
 
-For this reason, the Query Loop block supports a property called `allowedControls` which accepts an array of keys of the controls we want to display on the inspector sidebar. By default, we accept all the controls, but as soon as we provide an array to this property, we want to be specific and specify only the controls which are going to be relevant for us!
+For this reason, the Query Loop variations support a property called `allowedControls`, which accepts an array of keys of the controls we want to display on the inspector sidebar. By default, we accept all the controls, but as soon as we provide an array to this property, we want to specify only the controls which are going to be relevant for us!
 
-As of version 13.9, the following controls are available:
+As of GB version 14.2, the following controls are available:
```

Comment:
```suggestion
As of Gutenberg version 14.2, the following controls are available:
```

(I'd avoid shorthands which are very understandable to us, but could be somewhat esoteric to consumers of the docs. Unless “Gutenberg” is repeated too many times, I wouldn't go for “GB”).